atd-etl/socrata_export/socrata_export.py

```diff
@@ -96,6 +95,12 @@
         # Format records
         records = config["formatter"](data, config["formatter_config"])
 
+        # truncate the existing data set once we have records
+        if records and not is_truncate_complete:
+            print(f'Truncating {config["table"]} table...')
+            client.replace(config["dataset_uid"], [])
+            is_truncate_complete = True
```

Comment:
this should help avoid the situation where the ETL bombs and we end up with a truncated dataset 😅

Reply:
thanks for adding this! 🙌 this ETL doesn't fail often, but, when it does, the dashboard is 😵